bin/strata-client/src/extractor.rs

```diff
@@ -218,7 +218,8 @@ mod tests {
 
     #[tokio::test]
     async fn test_extract_deposit_requests() {
-        logging::init();
+        // FIXME this is absurd, why are we doing this here?
+        logging::init(logging::LoggerConfig::with_base_name("strata-client-tests"));
```

Comment:
It this calling `btcio::client` by any means?
If yes there's a lot of good logging happening there that is quite useful.